### ISSUE TITLE
Dynamic bundle sizes

### ISSUE THREAD

**Issue by canac** [transferwise/actions-next-bundle-analyzer#46]
It appears that between `master` and `v2` support for showing info about the dynamic bundle chunks was removed. Was that intentional? Would you be open to receiving a PR to add that back?

**Comment by Wise-StenRaudmets** [comment]
Yes it was removed. We found it was creating a lot of noise in the PR comments since we have dependencies which use dynamic imports internally quite a lot while our direct usage of dynamic imports was pretty minimal.
This means that whenever the dependency was updated or even the path of how a package manager like pnpm resolves the dependency was changed due to peer dependency adjustments in dependencies, it was considered a completely new set of dynamic bundle chunks that had no reference size to compare against. So for the v2 branch it was easier to just drop the feature.

But having thought about it, I'd be open to having it added back in if it ignored dynamic imports within the `node_modules` directory. Would that work for you?
